src/WebJobs.Script/Config/HostTelemetryInitializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    internal class HostTelemetryInitializer : ITelemetryInitializer
```

Comment:
I think we had some other issues (like logging the process id) that we could all resolve inside this initializer as well. I'd rename it to be more generic like `ScriptTelemetryInitializer`.